src/components/shared/CustomIconButton/index.tsx

```diff
@@ -1,4 +1,8 @@
 import { FC } from 'react';
 import { IconButton, IconButtonProps } from '@mui/material';
 
-export const CustomIconButton: FC<IconButtonProps> = ({ children, sx }) => <IconButton sx={sx}>{children}</IconButton>;
+export const CustomIconButton: FC<IconButtonProps> = ({ children, sx, onClick }) => (
```

Comment:
let's change interface to CustomIconButtonProps extends IconButtonProps {
icon: JSX.Element} and
 CustomIconButton: FC< CustomIconButtonProps > = ({ icon, ...props }) =>